src/sass/_settings-colors.scss

```diff
@@ -1,5 +1,9 @@
 $global-white:        #fff;
+$global-black:        #000;
 $global-background:   #efefef;
 $global-grey:         #d7d7d7;
+$global-mid-grey:     #cdcdcd;
 $global-dark-grey:    #333;
+$global-x-dark-grey:  #666;
 $global-orange:       #e95420;
+$global-transparent:  transparent;
```

Comment:
If this is a module we would be using on different projects should we add ```!default``` to these variables? Allows easier overrides if it's included in other projects.